src/components/Intro.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import image from "../images/img.PNG";
```

Comment:
please rename the img  and image in understandable format, Something like homeBanner.png